src/Wallabag/CoreBundle/Entity/Entry.php

```diff
@@ -266,7 +266,7 @@ public function setContent($content)
     {
         $this->content = $content;
         $this->readingTime = Tools::getReadingTime($content);
-        $this->domainName = parse_url($this->url, PHP_URL_HOST);
+        $this->domainName = Tools::getHostname(parse_url($this->url, PHP_URL_HOST));
```

Comment:
Regarding how you will search in the database, I don't think you need to remove the subdomain or the `www.`. Maybe you just need to remove the _too common_ `www.` but some subdomain are important. Like for Tumblr, you won't search for `tumblr` but using the subdomain of the tumblr, which is more accurate.